server/text_generation_server/utils/tokens.py

```diff
@@ -248,9 +260,14 @@ def __init__(
         grammars: List[str],
         grammar_types: List[int],
         fsm_grammar_states=List[int],
+        logit_biases: List[Optional[dict]] = None,
     ):
         warpers = []
 
+        # Initialize with empty logit biases if none provided
+        if logit_biases is None:
+            logit_biases = [None] * len(do_sample)
+
```

Comment:
Can we do like other arguments, and just send everything initialized instead of `None`

Reply:
yep that makes sense, updated in latest commit. Thanks!